sql/org_mozilla_firefox_derived/events_daily_v1/query.sql

```diff
@@ -0,0 +1,71 @@
+WITH events AS (
+  SELECT
+    submission_timestamp,
+    timestamp,
+    category,
+    name AS event,
+    extra AS event_properties,
+    client_info.* EXCEPT (os, os_version),
+    metadata.geo.city,
+    metadata.geo.country,
+    metadata.geo.subdivision1,
+    normalized_channel AS channel,
+    normalized_os AS os,
+    normalized_os_version AS os_version,
+    (
+      SELECT
+        ARRAY_AGG(STRUCT(key, value.branch AS value))
+      FROM
+        UNNEST(ping_info.experiments)
+    ) AS experiments,
+  FROM
+    org_mozilla_firefox.events,
+    UNNEST(events)
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+    OR (@submission_date IS NULL AND @submission_date >= '2020-01-01')
```

Comment:
The `OR (...)` part would always be false, no? Is it `DATE(submission_timestamp) >= '2020-01-01'`?